### PR TITLE
treatment phase2 updates

### PR DESCRIPTION
**Pull request by bullenc**

### CODE REVIEW

File: src/gdcdictionary/schemas/treatment.yaml [hunk 1]
```diff
@@ -160,6 +160,39 @@ properties:
     type: number
     minimum: 0
 
+  margin_distance:
+    $ref:
+      - "_terms.yaml#/margin_distance/common"
+    type: number
+    minimum: 0
```
Comment (ebusygina) [inline src/gdcdictionary/schemas/treatment.yaml:167]: should we add "minimum: 0" to the Jira task?

File: src/gdcdictionary/schemas/treatment.yaml [hunk 2]
```diff
@@ -160,6 +160,39 @@ properties:
...
+      - Involved
+      - Uninvolved
+      - Unknown
+      - Not Reported
+
+  margins_involved_site:
+    $ref:
+      - "_terms.yaml#/margins_involved_site/common"
+    enum:
+      - Cervix uteri
```
Comment (ebusygina) [inline src/gdcdictionary/schemas/treatment.yaml]: should "Cervix uteri" be capitalized? now it's like in the task

File: src/gdcdictionary/schemas/treatment.yaml (hunk 1 above)
Comment (bullenc) [reply src/gdcdictionary/schemas/treatment.yaml:167]: added 👍

File: src/gdcdictionary/schemas/treatment.yaml (hunk 2 above)
Comment (wwysoc2) [inline src/gdcdictionary/schemas/treatment.yaml]:
```suggestion
      - Cervix Uteri
```

**Review by wwysoc2 — CHANGES_REQUESTED** [review]
Added one suggestion, everything else looks good.

File: src/gdcdictionary/schemas/treatment.yaml (hunk 2 above)
Comment (bullenc) [reply src/gdcdictionary/schemas/treatment.yaml]: updated 👍

Comment (bullenc) [reply src/gdcdictionary/schemas/treatment.yaml]: updated 👍

**Review by ProfOak — APPROVED** [review]

File: src/gdcdictionary/schemas/treatment.yaml [hunk 3]
```diff
@@ -160,6 +160,39 @@ properties:
...
+    $ref:
+      - "_terms.yaml#/margin_status/common"
+    enum:
+      - Indeterminate
+      - Involved
+      - Uninvolved
+      - Unknown
+      - Not Reported
+
+  margins_involved_site:
```
Comment (stilesj-uchicago) [inline src/gdcdictionary/schemas/treatment.yaml:179]: Why the difference between 'margins' and 'margin' in the name?

**Review by stilesj-uchicago — APPROVED** [review]

File: src/gdcdictionary/schemas/treatment.yaml (hunk 3 above)
Comment (bullenc) [reply src/gdcdictionary/schemas/treatment.yaml:179]: I am not sure, this property is being migrated from another node, @tlicht3 would you know?

**Review by wwysoc2 — APPROVED** [review]

File: src/gdcdictionary/schemas/treatment.yaml [hunk 4]
```diff
@@ -160,6 +160,39 @@ properties:
...
+    $ref:
+      - "_terms.yaml#/margins_involved_site/common"
+    enum:
+      - Cervix Uteri
```
Comment (wteouchicago) [inline src/gdcdictionary/schemas/treatment.yaml:183]: the Jira has "uteri", but the PR looks right to me.
Edit: just saw Evgeniia's and Bill's comments. Maybe we just need to update the ticket?

**Review by wteouchicago — APPROVED** [review]

File: src/gdcdictionary/schemas/treatment.yaml (hunk 4 above)
Comment (bullenc) [reply src/gdcdictionary/schemas/treatment.yaml:183]: updated ticket 👍

**Pull request merged**